apps/api/lib/validations/destination-calendar.ts

```diff
@@ -14,8 +14,8 @@ const schemaDestinationCalendarCreateParams = z
   .object({
     integration: z.string(),
     externalId: z.string(),
-    eventTypeId: z.number(),
-    bookingId: z.number(),
+    eventTypeId: z.number().optional(),
+    bookingId: z.number().optional(),
     userId: z.number(),
```

Comment:
```suggestion
    userId: z.number().optional(),
```

Reply:
This should also be optional I think, or we wouldn't be able to have event type only destination calendars

Reply:
Also, need to validate the eventType belongs to the user making the request